### DISCUSSION TITLE
need how to build & use

### DISCUSSION THREAD

**Discussion by OpsCipher** [projectdiscovery/cdncheck#49] [Q&A]
hi we need to know how to use it or build the release show us some examples like always u do thank you

**Comment by ehsandeep** [comment]
@abramas the current release is used as a library in httpx and not a CLI tool, so there was no readme.

there is a CLI version of it implemented in the `dev` branch and details will be added upon next release.

**Comment by ehsandeep** [comment]
See the readme after latest release https://github.com/projectdiscovery/cdncheck
